### ISSUE TITLE
bold

### ISSUE THREAD

**Issue by omegatiger** [archriss/react-native-render-html#18]
```
<HTML html={'<div>Lorem Ipsun <b>Bold</b>  Foo <i>Italic</i>.</div>'} />
```

Bold and Italic text is breaking the line.

**Comment by Exilz** [comment]
Can you try adding the `bold` tag in here ? https://github.com/archriss/react-native-render-html/blob/master/HTMLUtils.js#L11

**Comment by Exilz** [comment]
This is fixed with the new parser with version `3.0.0`, I'll keep you posted.

**Comment by Exilz** [comment]
This is fixed with `3.0.0`, feel free to re-open this issue if you think it's not.

**Comment by omegatiger** [comment]
Tks!
